src/common/globalregex.cc

```diff
@@ -55,6 +55,9 @@ QRegularExpression Mdx::srcRe(
 QRegularExpression Mdx::srcRe2(
   R"(([\s"'](?:src|srcset)\s*=)\s*(?![\s"']|\b(?:(?:bres|https?|ftp)://|(?:data|javascript):))(?:file://)?[\x00-\x1f\x7f]*\.*/?([^\s">]+))",
   QRegularExpression::CaseInsensitiveOption );
+// matches srcset in <img srcset="text">
+QRegularExpression Mdx::srcset( R"((?<before>[\s]srcset\s*=\s*["'])\s*(?<text>[\s\S]*)(?<after>[\"']))",
```

Comment:
<after>[\"']
or
<after>["']

Reply:
It should be just `<after>["']` because `"` has no special meaning in PCRE.

I think only the symbols mentioned in this section needs to be escaped https://www.pcre.org/original/doc/html/pcrepattern.html#SEC4